packages/app-form-builder/package.json

```diff
@@ -19,7 +19,7 @@
     "@editorjs/editorjs": "^2.20.1",
     "@emotion/core": "^10.0.17",
     "@emotion/styled": "^10.0.17",
-    "@rmwc/menu": "^5.6.0",
+    "@rmwc/menu": "7.0.3",
```

Comment:
This bothers me because `@rmwc` packages should only be used in the `ui` package. Please make a note of these packages, and we should solve this in a separate PR. All our apps should be using the `ui` package exclusively, to maintain the abstraction in one place.

Reply:
Yep makes perfect sense